cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,20 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    add_definitions(-D_USE_MATH_DEFINES)
```

Comment:
Was this necessary for Axom?  Wondering if it should be placed in our axom_config, instead of here. 

Seems like it will be needed by client code. 

https://msdn.microsoft.com/en-us/library/4hwaceh6.aspx